apps/taxes/src/modules/taxes/tax-error.ts

```diff
@@ -0,0 +1,15 @@
+import { BaseError } from "../../error";
+
+const TaxError = BaseError.subclass("TaxError");
+
+// Error thrown when there is not enough data in webhook payload to proceed with the process. Is expected behavior. Not reported.
+export const TaxBadWebhookPayloadError = TaxError.subclass("TaxBadWebhookPayloadError");
```

Comment:
Maybe we could add some additional data in `props` so that we know this is expected error?

For example:

```ts
export const MissingSaleorApiUrlError = BaseError.subclass(`MissingSaleorApiUrlError`, {
  props: { sentrySeverity: "warning" } as CommonErrorProps,
});
```

Or like this:

```ts
props: { expected: true }
```

Reply:
Actualy both "expected" and "severity" is useful
1. Expected flag should help us with control flow, we should know what to respond to the client
2. Sentry severity can be controlled via env and we can fine tune it

Reply:
done